pyproject.toml

```diff
@@ -8,34 +8,33 @@ packages = [{ include = "scripts" }]
 
 [tool.poetry.dependencies]
 python = ">=3.9,<3.10"
-cairo-lang = "^0.11.0.1"
+cairo-lang = "^0.11"
 openzeppelin-cairo-contracts = "0.5.0"
 amarna = "^0.1.5"
-starknet-py = "^0.15.0"
+starknet-py = "^0.15"
 
 [tool.poetry.group.dev.dependencies]
 black = "^22.10.0"
 isort = "^5.10.1"
-cairo-coverage = { git = "https://github.com/starknet-edu/cairo-coverage.git", branch="master" }
+cairo-coverage = { git = "https://github.com/starknet-edu/cairo-coverage.git", branch = "master" }
```

Comment:
Just a note but `cairo-coverage` was published on pypi so we could just add it with `cairo-coverage = "^0.1.5"`